list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The code repeats the database query logic across different files.

Recommendations: Use a repository pattern to centralize database operations and reduce code duplication.

Example:
```php
class BookRepository {
    private $connection;

    public function __construct($connection) {
        $this->connection = $connection;
    }

    public function getAllBooks() {
        $query = "SELECT * FROM books";
        return $this->connection->query($query);
    }
}

// Usage
$bookRepo = new BookRepository($connection);
$result = $bookRepo->getAllBooks();
```